### PR TITLE
Use androidx.biometric.BiometricPrompt instead of android platform APIs

### PR DESCRIPTION
**Pull request by romandev**
This patch replaces the existing BiometricPrompt backend and
FingerprintManager backend with the androidx.biometric.BiometricPrompt.

Refs: #578

### CODE REVIEW

File: src/org/triple/banana/authentication/BiometricPromptActivity.java [hunk 1]
```diff
@@ -5,44 +5,36 @@
...
+import static androidx.biometric.BiometricPrompt.ERROR_LOCKOUT;
+import static androidx.biometric.BiometricPrompt.ERROR_LOCKOUT_PERMANENT;
 
-import android.annotation.SuppressLint;
-import android.annotation.TargetApi;
 import android.app.AlertDialog;
-import android.hardware.biometrics.BiometricPrompt;
 import android.os.Build;
 import android.os.Bundle;
 import android.os.CancellationSignal;
```
Comment (jdragonbae) [inline src/org/triple/banana/authentication/BiometricPromptActivity.java:14]: unused import

File: src/org/triple/banana/authentication/BiometricPromptActivity.java [hunk 2]
```diff
@@ -5,44 +5,36 @@
...
-import static android.hardware.biometrics.BiometricPrompt.BIOMETRIC_ERROR_LOCKOUT;
-import static android.hardware.biometrics.BiometricPrompt.BIOMETRIC_ERROR_LOCKOUT_PERMANENT;
+import static androidx.biometric.BiometricPrompt.ERROR_LOCKOUT;
+import static androidx.biometric.BiometricPrompt.ERROR_LOCKOUT_PERMANENT;
 
-import android.annotation.SuppressLint;
-import android.annotation.TargetApi;
 import android.app.AlertDialog;
-import android.hardware.biometrics.BiometricPrompt;
 import android.os.Build;
```
Comment (jdragonbae) [inline src/org/triple/banana/authentication/BiometricPromptActivity.java:12]: unused import

File: src/org/triple/banana/authentication/FingerprintManagerActivity.java [hunk 3]
```diff
@@ -1,135 +0,0 @@
...
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
-            finish();
-            return;
-        }
-
-        // Setting for finguerprint ui
-        setContentView(R.layout.fingerprint_dialog);
```
Comment (jdragonbae) [inline src/org/triple/banana/authentication/FingerprintManagerActivity.java:45]: Can we remove with `res/layout/fingerprint_dialog.xml`?

File: src/org/triple/banana/authentication/BiometricPromptActivity.java (hunk 1 above)
Comment (romandev) [reply src/org/triple/banana/authentication/BiometricPromptActivity.java]: Done

File: src/org/triple/banana/authentication/BiometricPromptActivity.java (hunk 2 above)
Comment (romandev) [reply src/org/triple/banana/authentication/BiometricPromptActivity.java]: Done

File: src/org/triple/banana/authentication/FingerprintManagerActivity.java (hunk 3 above)
Comment (romandev) [reply src/org/triple/banana/authentication/FingerprintManagerActivity.java:45]: Done

**Review by romandev — COMMENTED** [review]
@jdragonbae Thank you for review. I addressed your comments. Could you please check my new patch?

**Review by jdragonbae — APPROVED** [review]
LGTM

**Pull request merged**